### PR TITLE
Add data reduction rule

### PR DESCRIPTION
**Pull request by seattleplus**
Please review data reduction optimization rule.  I tried to make it short like the other rules, but happy to expand if necesssary.

### CODE REVIEW

**Review by gflofst — DISMISSED** [review]
I think this: "such as deduplication, compression, etc"
might be better as: "such as deduplication, compression, and other lossless and lossy techniques"
I am ok with as is though.

**Review by gmarkomanolis — DISMISSED** [review]
This reminds me again that if a system works like that and they have advanced techniques that improve performance is good, right? Yes, they can not be compared with other submissions that do not violate the rule but it would be nice to demonstrate them otherwise? Experimental list or something? I agree overall with the rule.

File: templates/Pages/rules_submission.php [hunk 1]
```diff
@@ -65,6 +65,14 @@
             data, it must ensure that data is persistently stored before
             acknowledging the close.
         </li>
+        <li>
+            Data and metadata must be written in its entirety and not reduced based
+            on its contents. The goal of the io500
+            is to provide dataset independent performance results, and techniques
+            such as deduplication, compression, etc would bias performance as the
+            io500 benchmark uses partially predictable content that is not realistic
```
Comment (JulianKunkel) [inline templates/Pages/rules_submission.php:73]: I would replace realistic with: "representative of existing workloads".

**Review by JulianKunkel — APPROVED** [review]

**Comment by JulianKunkel** [comment]
While I approved, I suppose we can discuss to change the term "realistic" which seems a bit harsh (as in my comment).

**Comment by gflofst** [comment]
I incorporated my and Julian's requested changes. Please review again.

**Review by JulianKunkel — DISMISSED** [review]
Could change io500 to IO500 for consistency but I'll approve.

**Review by seattleplus — COMMENTED** [review]
Great changes, thanks!

**Review by JulianKunkel — APPROVED** [review]

File: templates/Pages/rules_submission.php (hunk 1 above)
Comment (adilger) [reply templates/Pages/rules_submission.php]: I would write "dataset-independent", and "etc.", but not a huge deal.  I think someone else already pointed out "IO500".

**Review by adilger — APPROVED** [review]

**Review by gflofst — APPROVED** [review]

**Review by gmarkomanolis — APPROVED** [review]

**Pull request merged**